doc/conditional_independence.rst

```diff
@@ -0,0 +1,180 @@
+.. _conditional_independence:
+
+========================
+Conditional Independence
```

Comment:
I am wondering if simply "Independence" would be better. It currently sounds like that the unconditional case is a special case of the conditional independence simply with the conditioning set being empty. Although you can definitely see it like that, I would probably still distinguish between them more clearly. Why not just introduce independence tests in general and mention that one can either perform a unconditional or conditional test?

Reply:
Done. Lmk wdyt.